packages/backend/src/ee/models/AiAgentModel.ts

```diff
@@ -303,4 +303,25 @@ export class AiAgentModel {
             };
         });
     }
+
+    async deleteAgent({
+        organizationUuid,
+        agentUuid,
+    }: {
+        organizationUuid: string;
+        agentUuid: string;
+    }): Promise<void> {
+        const agent = await this.getAgent({
+            organizationUuid,
+            agentUuid,
+        });
```

Comment:
`getAgent` performs a big query, do we need to check the existence before deleting?
Asking because I'm not sure what error will the query return if the row is not found. Maybe we can avoid this extra request.